README.md

```diff
@@ -1,21 +1,235 @@
+|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:--------------:|
+|                                                                                                                                                                        <img src="src/main/resources/icons/unknown.svg" width="30">                                                                                                                                                                        |    Unknown     |
+|                                                                                                                                                                          <img src="src/main/resources/icons/low.svg" width="30">                                                                                                                                                                          |      Low       |
+|                                                                                                                                                                        <img src="src/main/resources/icons/medium.svg" width="30">                                                                                                                                                                         |     Medium     |
+|                                                                                                                                                                         <img src="src/main/resources/icons/high.svg" width="30">                                                                                                                                                                          |      High      |
+|                                                                                                                                                                       <img src="src/main/resources/icons/critical.svg" width="30">                                                                                                                                                                        |    Critical    |
+| <img src="src/main/resources/icons/unknownnotapplic.svg" height="15" width="15"><img src="src/main/resources/icons/lownotapplic.svg" height="15" width="15"><img src="src/main/resources/icons/mediumnotapplic.svg" height="15" width="15"><img src="src/main/resources/icons/highnotapplic.svg" height="15" width="15"><img src="src/main/resources/icons/criticalnotapplic.svg" height="15" width="15"> | Not Applicable |
```

Comment:
I think that the this graphic is too big. Let's make it smaller.
![image](https://user-images.githubusercontent.com/7830056/219960560-3abe0692-6e6d-4bf8-b16d-8c0d418069dd.png)

Let's also add a nore explaining what "Not Applicable" means.